spacelift/resource_azure_integration_test.go

```diff
@@ -15,14 +15,14 @@ func TestAzureIntegrationResource(t *testing.T) {
 
 	t.Run("Creates and updates an integration", func(t *testing.T) {
 		randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
-
 		testSteps(t, []resource.TestStep{
 			{
 				Config: fmt.Sprintf(`
 				resource "spacelift_azure_integration" "test" {
 					name      = "test-integration-%s"
 					tenant_id = "tenant-id"
 					labels    = ["one", "two"]
+					space_id  = "root"
```

Comment:
Really minor suggestion - maybe we should include test cases that don't require the space to be specified to ensure we haven't broken backwards compatibility?

Reply:
I have tested all the entities locally for both backwards comp and the new feature, so I am confident it's okay, but I don't mind adding more tests 👍

Reply:
Ok - up to you.